src/test/article.spec.js

```diff
@@ -668,3 +677,137 @@ describe('PUT /api/v1/articles/:articleId', () => {
     findAllStub.restore();
   });
 });
+
+describe('PUT /api/v1/articles/publish?articleId', () => {
+  beforeEach(async () => {
+    await db.Article.destroy({ cascade: true, truncate: true });
+    await db.User.destroy({ cascade: true, truncate: true });
+    await db.Follower.destroy({ cascade: true, truncate: true });
+  });
+  it('Returns 400 if the article Id is provided but is not a number', async () => {
+    const firstUser = await createUser();
+    const token = await helper.jwtSigner(firstUser);
+
+    const response = await server()
+      .put(`${ARTICLES_API}/publish?articleId=${'men'}`)
+      .set('x-access-token', token)
+      .send(article);
+    expect(response.status).to.equal(400);
+    expect(response.body.message).to.equal('articleId must be a number');
```

Comment:
You can say `Invalid article id` or to be verbose `Invalid article id. Article id must be a non-zero positive integer`